Contest_API.md

```diff
@@ -58,22 +58,22 @@ requested, e.g. at the URL path
 
 ` GET https://example.com/api/contests/wf14/teams`
 
-while an element with specific ID is requested as
+while an object with a specific ID is requested as
 
 ` GET https://example.com/api/contests/wf14/teams/10`
 
-A collection is always returned as a JSON list of objects. Every object
-in the list represents a single element (and always includes the ID).
-When requesting a single element the exact same object is returned. E.g.
+A collection is always returned as an array of JSON objects. Every item
+in the array is a single contest object (and always includes the ID).
```

Comment:
I understand that "contest object" is used here in a general way (i.e. one of the objects related to the contest), but a contest object could more specifically be the object returned from the contest endpoint. (This is exactly what we call it on line 584).

I suggest dropping "contest".

Reply:
Fixed